python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -29,6 +29,18 @@
 
 logger = Ganga.Utility.logging.getLogger(modulename=1)
 
+_imported_GangaList = None
+
+do_not_copy = ['_proxyObject', '_proxyClass', '_data', '_index_cache', '_parent', '_registry', '_impl']
+
+def _getGangaList():
+    global _imported_GangaList
+    if _imported_GangaList is None:
+        from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList
+        _imported_GangaList = GangaList
+    return _imported_GangaList
+
+
```

Comment:
(aside)
As well as this, the ```GangaList``` does modify a certain amount of ```Node``` and ```GangaObject``` functions which it would be ideal to have merged into the Objects class to some extent (although writing code to deeply inspect lists can be a pain)